main.go

```diff
@@ -275,6 +301,60 @@ func revokePermissions(dbName string) {
 	return
 }
 
+func restorePermissions(dbName string) {
+	db, err := sql.Open("mysql", dsn)
+	if err != nil {
+		log.Infof("Error opening connection to database: %s", err)
+		return
+	}
+	defer db.Close()
+	db.SetMaxOpenConns(3)
+	db.SetMaxIdleConns(1)
+	db.SetConnMaxLifetime(1 * time.Minute)
+
+	stmt, err := db.Prepare(
+		fmt.Sprintf(
+			"SELECT User, Host FROM mysql.db WHERE Db = '%s' AND (Insert_priv = 'N' OR Update_priv = 'N' OR Create_priv = 'N' OR Index_priv = 'N')",
+			dbName,
+		),
+	)
+	if err != nil {
+		panic(err.Error())
+	}
+	defer stmt.Close()
```

Comment:
since `stmt.Close()` is specifically called few lines below, this defer is not needed.